File: eApartman/eApartman.Model/ApartmanSlika.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eApartman.Model
{
    public class ApartmanSlika
    {
        public int SlikaId { get; set; }
        public byte[] SlikaFile { get; set; }
        public byte[] SlikaThumbnail { get; set; }
        public int? ApartmanId { get; set; }
    }
}

File: eApartman/UI/eApartman.MobileUI/lib/models/apartman_slika.dart
import 'dart:convert';

class ApartmanSlika {
  int slikaId;
  List<int> slikaFile;
  int apartmanId;

  ApartmanSlika({this.slikaFile, this.slikaId, this.apartmanId});

  factory ApartmanSlika.fromJson(Map<String, dynamic> json) {
    String stringBytes = json["slikaFile"] as String;
    List<int> bytes = base64.decode(stringBytes);
    return ApartmanSlika(
      slikaId: json["slikaId"],
      slikaFile: bytes,
      apartmanId: json["apartmanId"],
    );
  }
}
